packages/website/bin/page-generator/templates/index.js

```diff
@@ -1,382 +1,9 @@
-};
-
-const generateChangelogPage = (
-  pagePath,
-  changelogPath,
-  data,
-  config,
-  title = '',
-) => {
-  const componentPath = changelogPath;
-  const wrapperName = 'package-changelog';
-  const { wrappersPath, pagesPath } = config;
-
-  const {
-    componentImportPath,
-    packageHomeWrapperPath,
-    pageTitleComponentPath,
-  } = getGenericPageInfo(
-    pagesPath,
-    pagePath,
-    componentPath,
-    wrappersPath,
-    wrapperName,
-  );
-
-  writeFile(
-    path.join(pagesPath, pagePath),
-    changelogTemplate(
-      componentImportPath,
-      packageHomeWrapperPath,
-      pageTitleComponentPath,
-      data,
-      title,
-    ),
-  );
-};
-
-const generatePackageDocPage = (
-  pagePath,
-  markdownPath,
-  data,
-  config,
-  title = '',
-) => {
-  generateBasicPage(
-    pagePath,
-    markdownPath,
-    data,
-    'package-docs',
-    config,
-    title,
-  );
-};
-
-const generateExamplePage = (
-  pagePath,
-  rawPagesPath,
-  exampleModulePath,
-  data,
-  config,
-  title = '',
-) => {
-  const componentPath = exampleModulePath;
-  const wrapperName = 'package-example';
-  const { wrappersPath, pagesPath } = config;
-
-  const {
-    componentImportPath,
-    packageHomeWrapperPath,
-    pageTitleComponentPath,
-  } = getGenericPageInfo(
-    pagesPath,
-    pagePath,
-    componentPath,
-    wrappersPath,
-    wrapperName,
-  );
-
-  writeFile(
-    path.join(pagesPath, pagePath),
-    exampleTemplate(
-      componentImportPath,
-      packageHomeWrapperPath,
-      pageTitleComponentPath,
-      data,
-      title,
-    ),
-  );
-
-  writeFile(
-    path.join(pagesPath, rawPagesPath),
-    rawTemplate(path.join('..', componentImportPath)),
-  );
-};
-
-const generateDocsHomePage = (pagePath, data, config, title = '') => {
-  generateNonComponentPage(pagePath, data, 'item-list', config, 'docs', title);
-};
-
-const generateExamplesHomePage = (pagePath, data, config, title = '') => {
-  generateNonComponentPage(
-    pagePath,
-    data,
-    'item-list',
-    config,
-    'examples',
-    title,
-  );
-};
-
-const generateProjectDocPage = (
-  pagePath,
-  markdownPath,
-  data,
-  config,
-  title = '',
-) => {
-  generateBasicPage(
-    pagePath,
-    markdownPath,
-    data,
-    'project-docs',
-    config,
-    title,
-  );
-};
-
-module.exports = {
-  generateHomePage,
-  generateChangelogPage,
-  generatePackageDocPage,
-  generateExamplePage,
-  generateDocsHomePage,
-  generateExamplesHomePage,
-  generateProjectDocPage,
-};
+const changelogTemplate = require('./changelog');
+const exampleTemplate = require('./example');
+const singleComponentTemplate = require('./single-component');
+const wrappedComponentTemplate = require('./wrapped-component');
+
+module.exports.changelogTemplate = changelogTemplate;
+module.exports.exampleTemplate = exampleTemplate;
+module.exports.singleComponentTemplate = singleComponentTemplate;
+module.exports.wrappedComponentTemplate = wrappedComponentTemplate;
```

Comment:
What's the reason for changing the syntax here?

Reply:
tl;dr I don't know which is better and i just picked one